use-case-guides/workday-request-time-off/README.md

```diff
@@ -1,131 +1,104 @@
+2. Install the Custom Report below in Workday
 
-## Configuration Instructions
+    Use the custom report definitions below to install the columns and prompt defaults for your custom report. This sets up calculated fields for the time off request and time off balance which are required for the plugin to work.
 
-### Overview
+    [Worker Details Moveworks Columns.xlsx](https://developer.moveworks.com/file-hosting/workday/Worker_Details_Moveworks_Columns.xlsx)
 
-We will be accessing Workday through the read-only WQL API and the read + write AbsenceManagement REST API, through a Workday Connector in Moveworks. This requires you to:
+    [Worker Details Moveworks Prompt.xlsx](https://developer.moveworks.com/file-hosting/workday/Worker_Details_Moveworks_Prompt.xlsx)
 
-1. Create a Connector in Creator Studio – [link](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday)
-   * Grant the [domain permissions below](#domain-permissions-required) as part of your Integrations Security Systems Group (ISSG)
-2. Test required APIs in Postman
-3. Build use case in Creator Studio
+3. Share your Workday API credentials with your Moveworks developer team.
 
-### Domain Permissions Required
+## Domain Permissions Required
 
 | Workday Web Service | Domain Permissions Required | Description |
 | --- | --- | --- |
 | Worker Data Query (WQL) | Get_References, Get_Workers, Get_Eligible_Time_Off_Plans, Get_Time_Off_Balance, Get_Positions | Permissions needed to retrieve worker information, including contact details, position, and time off balances. |
-| Time Off Request (ReST) | Put_Time_Off_Request, Get_Workers, Get_Positions | Permissions needed to submit time off requests and retrieve worker and position information. |
-
-
-> ℹ️ **You will be creating two ISUs, one in a staging environment, and one in production.**
+| Time Off Request (REST) | Put_Time_Off_Request, Get_Workers, Get_Positions | Permissions needed to submit time off requests and retrieve worker and position information. |
 
+![Domain%20Permissions](Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/Screenshot%202024-07-12%20at%2012.49.29 PM.png)
 
-### Step 1: Create a Connector in Creator Studio
+**Inform your Creator Studio developer that they can now use your Workday credentials to build their connector and plugin**.
 
-[Follow the authentication guide to create a working Workday connector in Creator Studio](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday)
 
-### **Step 2: Test required APIs with Postman**
+# For Creator Studio Developers
 
-**Prerequisite: Install Custom Report in Workday**
+## Step 1: Validate you have built a Creator Studio Connector for your middleware / APIM tool
```

Comment:
What's our style guide around "APIM" vs. "Middleware" vs. "iPaaS" -- feels like the org has gravitated around using the word "iPaaS"

Reply:
Although middleware may be appropriate and a more generic term

Reply:
Use middleware